src/views/Swap/index.tsx

```diff
@@ -377,8 +376,8 @@ export default function Swap({ history }: RouteComponentProps) {
                   setIsChartDisplayed={setIsChartDisplayed}
                   isChartDisplayed={isChartDisplayed}
                 />
-                <Wrapper id="swap-page">
-                  <AutoColumn gap="md">
+                <Wrapper id="swap-page" style={{ minHeight: '412px' }}>
```

Comment:
minHeight is only for `Insufficient liquidity for this trade.Try enabling multi-hop trades.` error. Other that this it behaves like fixed height